addon/components/flash-message.js

```diff
@@ -107,5 +107,11 @@ export default Component.extend({
     if (flash) {
       flash.destroyMessage();
     }
+  },
+
+  actions: {
+    close() {
+      this._destroyFlashMessage();
```

Comment:
What do you think of adding a warning via `Ember.Logger.warn` if `flash.destroyOnClick` is true?  Something like:

```
close() {
  if (get(this, 'flash.destroyOnClick')) {
    Logger.warn('This flash message has `destroyOnClick` enabled -- be sure to disable it when using the close action')
  }
  this._destroyFlashMessage();
}
```